docs/guides/payments.rst

```diff
@@ -0,0 +1,236 @@
+            token=response.transaction.id,
+            is_success=response.is_success,
+            gateway_response=get_gateway_response(response))
+        return txn, response['error']
+
+capture(payment, amount, **connection_params)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A transfer of the money that was reserved during the authorization stage.
+
+Example
+"""""""
+
+.. code-block:: python
+
+    def capture(
+            payment: Payment,
+            amount: Decimal,
+            **connection_params: Dict) -> Tuple[Transaction, str]:
+
+        # Please note that token from the last AUTH transaction should be used
+        auth_transaction = payment.transactions.filter(
+            transaction_type=TransactionType.AUTH).first()
+        transaction_token = auth_transaction.token
+
+        # Handle connecting to the gateway and sending the capture request here
+        response = gateway.capture(token=transaction_token)
+
+        txn = create_transaction(
+            payment=payment,
+            transaction_type=TransactionType.CAPTURE,
+            amount=amount,
+            currency=payment.currency,
+            token=response.transaction.id,
+            is_success=response.is_success,
+            gateway_response=get_gateway_response(response))
+        return txn, response['error']
+
+void(payment, **connection_params)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A cancellation of a pending authorization or capture.
+
+Example
+"""""""
+
+.. code-block:: python
+
+    def void(
+            payment: Payment,
+            **connection_params: Dict) -> Tuple[Transaction, str]:
+
+        # Please note that token from the last AUTH transaction should be used
+        auth_transaction = payment.transactions.filter(
+            transaction_type=TransactionType.AUTH).first()
+        transaction_token = auth_transaction.token
+
+        # Handle connecting to the gateway and sending the void request here
+        response = gateway.void(token=transaction_token)
+
+        txn = create_transaction(
+            payment=payment,
+            transaction_type=TransactionType.VOID,
+            amount=payment.total,
+            currency=payment.currency,
+            gateway_response=get_gateway_response(response),
+            token=response.transaction.id,
+            is_success=response.is_success)
+        return txn, response['error']
+
+Parameters
+^^^^^^^^^^
++-----------------------+-----------+------------------------------------------------------------------------------------------------+
+| name                  | type      | description                                                                                    |
++-----------------------+-----------+------------------------------------------------------------------------------------------------+
+| ``payment``           | `Payment` | Payment instance, for which the transaction will be created.                                   |
```

Comment:
Not sure if that accidental or not, but wouldn't it be better to put it in an inline code?

Below is a patch.
```patch
Index: docs/guides/payments.rst
<+>UTF-8
===================================================================
--- docs/guides/payments.rst	(revision a4f77a9d22216905738c6442867871f0b48ea1fb)
+++ docs/guides/payments.rst	(date 1540674255000)
@@ -167,29 +167,29 @@
 
 Parameters
 ^^^^^^^^^^
-+-----------------------+-----------+------------------------------------------------------------------------------------------------+
-| name                  | type      | description                                                                                    |
-+-----------------------+-----------+------------------------------------------------------------------------------------------------+
-| ``payment``           | `Payment` | Payment instance, for which the transaction will be created.                                   |
-+-----------------------+-----------+------------------------------------------------------------------------------------------------+
-| ``transaction_token`` | `str`     | Unique transaction's token that will be used on the purpose of completing the payment process. |
-+-----------------------+-----------+------------------------------------------------------------------------------------------------+
-| ``connection_params`` | `dict`    | List of parameters used for connecting to the payment's gateway.                               |
-+-----------------------+-----------+------------------------------------------------------------------------------------------------+
-| ``amount``            | `Decimal` | Amount of Money to be refunded/captured.                                                       |
-+-----------------------+-----------+------------------------------------------------------------------------------------------------+
++-----------------------+-------------+------------------------------------------------------------------------------------------------+
+| name                  | type        | description                                                                                    |
++-----------------------+-------------+------------------------------------------------------------------------------------------------+
+| ``payment``           | ``Payment`` | Payment instance, for which the transaction will be created.                                   |
++-----------------------+-------------+------------------------------------------------------------------------------------------------+
+| ``transaction_token`` | ``str``     | Unique transaction's token that will be used on the purpose of completing the payment process. |
++-----------------------+-------------+------------------------------------------------------------------------------------------------+
+| ``connection_params`` | ``dict``    | List of parameters used for connecting to the payment's gateway.                               |
++-----------------------+-------------+------------------------------------------------------------------------------------------------+
+| ``amount``            | ``Decimal`` | Amount of Money to be refunded/captured.                                                       |
++-----------------------+-------------+------------------------------------------------------------------------------------------------+
 
 Returns
 ^^^^^^^
-+-----------------------+---------------+-----------------------------------------------------------------------------------------------------------+
-| name                  | type          | description                                                                                               |
-+-----------------------+---------------+-----------------------------------------------------------------------------------------------------------+
-| ``txn``               | `Transaction` | Transaction created during the payment process, with ``is_success`` set to ``True`` if no error occurred. |
-+-----------------------+---------------+-----------------------------------------------------------------------------------------------------------+
-| ``error``             | `str`         | Error message to be displayed in the UI, empty if no error occurred.                                      |
-+-----------------------+---------------+-----------------------------------------------------------------------------------------------------------+
-| ``transaction_token`` | `str`         | Unique transaction's token that will be used on the purpose of completing the payment process.            |
-+-----------------------+---------------+-----------------------------------------------------------------------------------------------------------+
++-----------------------+-----------------+-----------------------------------------------------------------------------------------------------------+
+| name                  | type            | description                                                                                               |
++-----------------------+-----------------+-----------------------------------------------------------------------------------------------------------+
+| ``txn``               | ``Transaction`` | Transaction created during the payment process, with ``is_success`` set to ``True`` if no error occurred. |
++-----------------------+-----------------+-----------------------------------------------------------------------------------------------------------+
+| ``error``             | ``str``         | Error message to be displayed in the UI, empty if no error occurred.                                      |
++-----------------------+-----------------+-----------------------------------------------------------------------------------------------------------+
+| ``transaction_token`` | ``str``         | Unique transaction's token that will be used on the purpose of completing the payment process.            |
++-----------------------+-----------------+-----------------------------------------------------------------------------------------------------------+
 
 Adding new payment provider in the settings
 -------------------------------------------
```

Reply:
Seems better, thanks